react/components/BuyButton/index.js

```diff
@@ -173,7 +173,7 @@ export const BuyButton = ({
       ) : (
         <Button
           block={large}
-          disabled={disabled || !available}
+          disabled={disabled || !available || orderFormContext.loading}
```

Comment:
I think it's better to active the prop `isLoading` instead

Reply:
@klzns I agree that sounds better, but the UX will be weird.
When load the page, the user will see the button spinning, even thought he don't clicked, I don't think this behavior is expected.
What do you think?